src/webrtc/vad.md

```diff
@@ -0,0 +1,76 @@
+# Active Speaker Detection
+
+A common feature in video conference software is active speaker detection. It is a small feature but one that the user would feel weird without. Oftentimes, it takes the form of a gentle highlight of the user's tile when they are talking.
+Other use cases include deciding which participants should be shown on the screen.
+This can be implemented eg. using historical data about their voice activity.
+
+As you can see, voice activity detection is a small but important piece of any WebRTC implementation.
+WebRTC standard provides tools that make it easy and convenient to implement in your Selective Forwarding Unit.
+
+## Audio Level Header extension
+RFC 6464 defines an Audio Level Header extension, very useful in the context of Voice Activity Detection, as it takes the implementation load of the SFU.
+
+The Extension carries the information about the audio level in `-dBov` with values from 0 to 127. Please mind the minus, it makes it so that the louder it gets, the lower the value.
+The fact that values are negative is a consequence of a definition of the `dBov` unit.
+
+RFC 6464 also defines an optional flag bit "V". When the use of it is negotiated, it indicates whether the encoder believes the audio packet contains voice activity.
+
+### What is `dBov`
+`dBov` is defined as the level, in decibels, relative to the overload point of the system.
+In this context, an overload point is a highest-intensity signal encodable by the payload format.
+
+All well and good, but how that translates into something readable? Well, let's start with "highest-intensity signal encodable by the payload format".
+It's simply the loudest volume that you could encode into the given codec, usually OPUS.
+
+Then there is an issue of decibels.
+**!!WARNING!! Physics intensifies**
```

Comment:
This is not displayed in a separate line. Did you run mdbook :thinking: ?

Reply:
Yes, it wasn't supposed to be displayed in a separate line. Actually, we might consider removing this line :stuck_out_tongue: